test/functional/collection/suite_test.go

```diff
@@ -4,10 +4,12 @@ import (
 	"testing"
 
 	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
 	. "github.com/onsi/gomega"
 )
 
 func TestLogForwarding(t *testing.T) {
 	RegisterFailHandler(Fail)
+	config.DefaultReporterConfig.SlowSpecThreshold = 120
```

Comment:
This is configurable as a switch in the make file

Reply:
Yes. Reason I did this is some tests don't understand ginkgo flags so if you 'go test' a group of tests that includes non-ginkgo tests with the ginkgo flag, it fails. On thing I'm aiming for is to be able to say "go test ..." with whatever set of tests you want. If this bothers you I can look for a different way to do it.

Reply:
I think generally "go test" with ginkgo is always a local dev problem.  To me this means, you dev, just need to understand the implications of "slow test" and we strive to solve by adding a higher level flag in the makefile.  That being said, I'm not hard pressed for you to remove this; it was just a suggestion to resolve not needing to touch so many files.